### PR TITLE
[CICD] Add Integration Test for gov module

### PR DESCRIPTION
**Pull request by yzang2019**
## Describe your changes and provide context
Changes:
1. Add integration test for normal param change proposal
2. Add integration test for expedited proposal

## Testing performed to validate your change
Test passed locally

### CODE REVIEW

File: integration_test/gov_module/gov_proposal_test.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+- name: Test making a new param change proposal should pass and take effect
+  inputs:
+    # Get the current tally params
+    - cmd: seid q gov params --output json |jq -r .tally_params.quorum
+      env: OLD_PARAM
```
Comment (BrandonWeng) [inline integration_test/gov_module/gov_proposal_test.yaml]:
```suggestion
    - cmd: seid q gov params --output json | jq -r .tally_params.quorum
      env: OLD_PARAM
```

File: integration_test/gov_module/gov_proposal_test.yaml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    # Make a new proposal
+    - cmd: printf "12345678\n" | seid tx gov submit-proposal param-change ./integration_test/gov_module/proposal/param_change_proposal.json 
+           --from admin --chain-id sei --fees 2000usei -b block -y --output json |jq -M -r ".logs[].events[].attributes[0] | select(.key == \"proposal_id\").value"
+      env: PROPOSAL_ID
+    # Get proposal status
+    - cmd: seid q gov proposal $PROPOSAL_ID --output json | jq -r .status
+      env: PROPOSAL_STATUS
+    # Make a deposit
+    - cmd: printf "12345678\n" | seid tx gov deposit $PROPOSAL_ID 10000000usei --from admin --chain-id sei --fees 2000usei -b block -y --output json |jq -r .code
```
Comment (BrandonWeng) [inline integration_test/gov_module/gov_proposal_test.yaml]:
```suggestion
    - cmd: printf "12345678\n" | seid tx gov deposit $PROPOSAL_ID 10000000usei --from admin --chain-id sei --fees 2000usei -b block -y --output json | jq -r .code
```

File: integration_test/gov_module/gov_proposal_test.yaml [hunk 3]
```diff
@@ -0,0 +1,73 @@
+- name: Test making a new param change proposal should pass and take effect
```
Comment (BrandonWeng) [inline integration_test/gov_module/gov_proposal_test.yaml:1]: Can we also add integration tests for the following failure scenarios where the coins are burned?

1. https://github.com/sei-protocol/sei-cosmos/blob/main/x/gov/keeper/tally.go#LL98C1-L104C3 
2. https://github.com/sei-protocol/sei-cosmos/blob/main/x/gov/keeper/tally.go#L111-L114 

Perhaps also the failure scenarios where its not burned too

**Review by philipsu522 — APPROVED** [review]

File: integration_test/gov_module/gov_proposal_test.yaml (hunk 3 above)
Comment (yzang2019) [reply integration_test/gov_module/gov_proposal_test.yaml:1]: Good idea!

**Pull request merged**